chatbot-ui/.eslintrc.json

```diff
@@ -1,3 +1,21 @@
 {
-  "extends": ["next/core-web-vitals", "plugin:prettier/recommended"]
+  "extends": [
+    "plugin:@typescript-eslint/recommended",
+    "plugin:react/recommended",
+    "plugin:react-hooks/recommended",
+    "plugin:jsx-a11y/recommended",
+    "plugin:import/typescript",
+    "prettier",
+    "prettier/prettier"
+  ], 
```

Comment:
I added "plugin:prettier/recommended" following the docs https://github.com/prettier/eslint-plugin-prettier to configure prettier within eslint. I'm just wondering with this change if prettier is still running with eslint. If not we might need to add a task into the pipeline to run a check with prettier.